src/Propel/Generator/Model/ScopedMappingModel.php

```diff
@@ -119,6 +125,22 @@ public function isAbsoluteNamespace($namespace)
         return strpos($namespace, '\\') === 0;
     }
 
+    /**
+     * Prepends a backslash to a namespace if there is none.
+     *
+     * A namespace with a backslash is considered absolute.
+     *
+     * @param string|null $namespace
+     *
+     * @return string|null
+     */
+    public static function makeNamespaceAbsolute(?string $namespace): ?string
```

Comment:
Why this method is static? As I can see we use it non static methods, so I guess we can make it protected and nonstatic

Reply:
Why would you make a method that does not access object properties nonstatic?

Reply:
Wouldnt a normal method and `$this->makeNamespaceAbsolute()` call suffice, is I think, here the question.

Reply:
Are you saying you want this to be an instance method, even when it does not work with the instance?
I am curious to know why, particularly if there is architectural concern behind it, or if you just like one better.
Either way, I am happy to change it, if it suits you better.

Reply:
Instance and protected invocation are always preferred over public and static.
The question at the beginning was for a reason on the latter. Otherwise, yes, let's use protected normal.

Reply:
Wait, are we talking about visibility (public vs protected) or association (class vs instance)?
Nevermind, I changed it to protected and nonstatic.